js/layout-javascript/agenda-code.js

```diff
@@ -203,6 +203,10 @@ DynamicList.prototype.attachObservers = function() {
       $(this).parents('.new-agenda-search-filter-overlay').removeClass('display');
       _this.$container.find('.section-top-wrapper, .agenda-cards-wrapper, .dynamic-list-add-item').removeClass('hidden');
 
+      if (!_this.$container.find('.clear-filters').hasClass('hidden')) {
```

Comment:
What exactly is this `if` clause logic looking to confirm?

Reply:
@tonytlwu I came up with such an output in order to see if we have selected filters, because at the moment the Appli button is pressed, there are no active filters in the variable `_this.activeFilters` and the same for filters in active filters container, so this is the way how i can fix this bug

Reply:
@AndrRyaz If you need to check if we have selected filters, then check if we have selected filters.

Something like `this.$container.find('.hidden-filter-controls-filter.mixitup-control-active')` should be able to check what's selected.

Reply:
@tonytlwu fixed